package.json

```diff
@@ -59,6 +60,7 @@
     "classnames": "^2.2.0",
     "history": "^1.13.0",
     "install": "^0.2.6",
+    "intl": "^1.0.1",
```

Comment:
Keeping `^` in your `package.json` is a dangerous way of life :punch:. You should use `~` or even better `--save-exact` option when running npm install to fix the version.